src/utils/getResult.js

```diff
@@ -0,0 +1,35 @@
+import { MissionUtils } from "@woowacourse/mission-utils";
+
+/**
+ * 컴퓨터의 숫자와 플레이어의 숫자를 비교하여 스트라이크와 볼을 얻어냅니다.
+ * @param {Object} params
+ * @param {number[]} params.computerNums - 컴퓨터의 숫자 배열
+ * @param {number[]} params.userNums - 플레이어가 입력한 숫자 배열
+ * @returns {{ strike: number, ball: number }} - 스트라이크와 볼 개수
+ */
+export async function getResult({ computerNums, userNums }) {
```

Comment:
```suggestion
export async function getResult({ computerNums, userNums }) {
```
`getResult` 함수 내부에 비동기 함수가 없는 것 같은데 `async` 함수로 선언하신 이유가 있으신가요??

Reply:
ㅋ.... 그러게요ㅠㅠ 코드를 이리저리 수정하다가 놓친것 같습니다ㅠ 감사합니다!